hack/lib/images.bash

```diff
@@ -235,6 +245,21 @@ function latest_konflux_image_sha() {
   echo "${image_without_tag}@${digest}"
 }
 
+function latest_registry_ci_sha() {
+  input=${1:?"Provide image"}
+
+  image_without_tag=${input%:*} # Remove tag, if any
+
+  go_bin="$(go env GOPATH)/bin"
+  export GOPATH="$PATH:$go_bin"
```

Comment:
not sure, why this is needed?

Reply:
I'm not sure. I copied it from the other functions in this file. Looks like it works without this as well. Maybe @pierDipi knows?

Reply:
Removed the line in the new commit.

Reply:
not everyone has the go/bin folder in their path and `go install` installs binaries in go/bin but I'm ok making it a prerequisite